Sources/TuistLoader/Models/XCTestPlan.swift

```diff
@@ -0,0 +1,49 @@
+struct XCTestPlan: Decodable {
```

Comment:
I'd import `Foundation` here. Is there a reason why this file is not in the `TuistGraph` target where all the other modules live?

Reply:
I'd also add coddle tests like we are doing with the other models.

Reply:
This is an internal object representing the JSON schema for `.xctestplan` files. It's used exclusively within `TuistLoader` (which loads it from the disk), and then `TuistLoader` uses `TuistGraph`'s `TestPlan` to build a Tuist-representation of the test plan (see the file right above).  
Let me know if that makes sense to you, I can add some comments to clarify if needed!

There is discussion to improve that so Test Plan are generated by Tuist directly (so it wouldn't need to be loaded anymore) [here](https://github.com/tuist/tuist/discussions/5214), and I think this should be done eventually but also think it's out of scope for this PR.